felix/vxlanfdb/vxlan_fdb.go

```diff
@@ -107,6 +108,12 @@ func WithNetlinkHandleShim(newNetlinkHandle func() (netlinkshim.Interface, error
 	}
 }
 
+func WithARPUpdatesOnly() Option {
```

Comment:
```suggestion
func WithNeighUpdatesOnly() Option {
```
IPv6 uses NDP so maybe we should be more generic here.